sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
 };
 use std::{cmp::Ordering, collections::BinaryHeap, iter::FromIterator, ops::Deref};
 
-pub const MAX_ENTRIES: usize = 150;
+pub const MAX_ENTRIES: usize = 300;
```

Comment:
Can we not define this relative to the actual `MAX_RECENT_BLOCKHASHES` const, so they don't get out of sync?

Reply:
There's a test that will break if `MAX_RECENT_BLOCKHASHES` exceeds `MAX_ENTRIES`. Idea being to detect ABI breakage and that making `MAX_RECENT_BLOCKHASHES` smaller is fine

Reply:
I still think this is unnecessarily brittle.
Also, why is making `MAX_RECENT_BLOCKHASHES` smaller but not changing the size of this list fine?

Reply:
Hmm yeah I suppose the semantics do break even though ABI is fine with a smaller list...

I feel like we need to make some noise here whenever `MAX_RECENT_BLOCKHASHES` changes.  Just to be sure the author is aware of this side effect.  Any suggestions?

Reply:
I don't quite see the need, since changing `MAX_RECENT_BLOCKHASHES` will be an abi change in the Bank anyway. But I might be missing something?

If you think it's important to have an indicator here, perhaps put the golden in the test instead of the const defintion with descriptive comments?

Reply:
Updated